### ISSUE TITLE
[Documenation] Run file for V3

### ISSUE THREAD

**Issue by bearrito** [just-containers/s6-overlay#515]
Hello,

tl;dr Proper way to exec/run a command in a RUN file

This is a basic question, but I'm unable to find a clear answer. It seems like most of the examples on the web are from a few years ago and/or aren't using the new method for defining services. I see mixed usage of exec, execlineb, setuid etc. Not sure what is valid for what use case. 

I've defined two services. I'm fairly sure they are laid out correctly under `/etc/s6-overlay/s6-rc.d`

My first RUN file is as below. 
This service will natively stay in the foreground
```
teleport start -c  /etc/teleport.yaml
```

Second run file. Runs in foreground
```
/app/simplewebsocket.py
```
I'm marking each run file as executable
```
COPY --chmod=777 resources/simplewebsocket/run /etc/s6-overlay/s6-rc.d/simplewebsocket/run
```

I seem to be missing something though...
```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service teleport: starting
s6-rc: info: service simplewebsocket: starting
s6-supervise simplewebsocket (child): fatal: unable to exec run: Exec format error
s6-supervise simplewebsocket: warning: unable to spawn ./run - waiting 10 seconds
s6-supervise teleport (child): fatal: unable to exec run: Exec format error
s6-supervise teleport: warning: unable to spawn ./run - waiting 10 seconds
```

If I missed a relevant portion in the docs somewhere please let me know.

**Comment by skarnet** [comment]
It looks like you're simply missing a shebang in your run files, to tell the kernel what kind of executables they are.

Add this line at the top of your run files:
```
#!/command/execlineb -S1
```
and you should be all good.

**Comment by skarnet** [comment]
Did that solve your issue? Can I close this?
